migrate/loader.go

```diff
@@ -0,0 +1,137 @@
+package migrate
+
+import (
+	"fmt"
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+	"os"
+	"strings"
+)
+
+var loaderCommand = &cobra.Command{
+	Use:   "loader",
+	Short: "Migrate your loader versions to newer versions.",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		modpack, err := core.LoadPack()
+		if err != nil {
+			// Check if it's a no such file or directory error
+			if os.IsNotExist(err) {
+				fmt.Println("No pack.toml file found, run 'packwiz init' to create one!")
+				os.Exit(1)
+			}
+			fmt.Printf("Error loading pack: %s\n", err)
+			os.Exit(1)
+		}
+		// Get our current loader
+		currentLoader := modpack.GetLoaders()
```

Comment:
Could you make this variable name plural to indicate that it is a slice.. or make another variable with `currentLoaders[0]` after the sanity check?